pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -465,6 +465,13 @@ func (r *ReconcileLogStorage) Reconcile(request reconcile.Request) (reconcile.Re
 		dexCfg = render.NewDexRelyingPartyConfig(authentication, dexTLSSecret, dexSecret, r.clusterDomain)
 	}
 
+	var oidcUserConfigMap *corev1.ConfigMap
+	var oidcUserSecret *corev1.Secret
+	if managementClusterConnection == nil {
+		oidcUserConfigMap = r.getOIDCUserConfigMapAvailable(ctx)
```

Comment:
Is there a watch for this resource?

Reply:
There is no watch on this ConfigMap or Secret, they should be created or deleted depending on es license type and dex config, we already have watch on them, also data in this ConfigMap & Secret will be updated by other components (es-proxy and kube-controller) so we don't have to watch for changes.

Reply:
I think we still need watches for them.
Unless it is guaranteed that the ConfigMap and Secret will be updated before the license type and dex config are updated. If not then it would be possible that the controller gets an update (from the watch) for ES license type and dex config, logstorage controller Reconciles, then the ConfigMap and Secret are updated. Logstorage would have missed the updates to the ConfigMap and Secret.